R/data.R

```diff
@@ -0,0 +1,39 @@
+#' @title Iran Census Population Counts
+#'
+#' @description Publicly available sex and age specific population counts for
```

Comment:
Add link for cite where these counts can be accessed? or a citation?

Reply:
Also spent some time messing around with including references but can't figure it out